src/de/cuuky/varo/configuration/configurations/config/ConfigSetting.java

```diff
@@ -20,9 +21,11 @@ public enum ConfigSetting implements SectionEntry {
 	// AUTOSETUP
 	AUTOSETUP_ENABLED(ConfigSettingSection.AUTOSETUP, "enabled", false, "Wenn Autosetup aktiviert ist, werden beim\nStart des Servers alle Spawns automatisch gesetzt und\noptional ein Autostart eingerichtet."),
 	AUTOSETUP_LOBBY_ENABLED(ConfigSettingSection.AUTOSETUP, "lobby.enabled", true, "Ob eine Lobby beim AutoSetup gespawnt werden soll"),
-	AUTOSETUP_LOBBY_HEIGHT(ConfigSettingSection.AUTOSETUP, "lobby.height", 10, "Hoehe der Lobby, die gespawnt werden soll"),
+	AUTOSETUP_LOBBY_SNAP(ConfigSettingSection.AUTOSETUP, "lobby.snap", AutoSetup.LobbySnap.MAX_HEIGHT.name(), "Naeherungsweise von welcher Position auf der Y-Achse\ndie Lobby gespawnt werden soll\n\nMoegliche Werte: GROUND, MAX_HEIGHT, ABSOLUTE"),
+	AUTOSETUP_LOBBY_SNAP_OFFSET(ConfigSettingSection.AUTOSETUP, "lobby.snapOffset", 50, "Wie weit von ihrem Snap-Punkt entfernt\ndie Lobby gespawnt werden soll"),

```

Comment:
The description should clarify this setting's behavior when using different options for `lobby.snap`. Or maybe there should a different config entry for each snap option (?)

Reply:
What exactly do you whish more clarity about in the description?  I described the values as positions, so it should be clear that the lobby is spawned at this position. If I were to really describe this in detail, the space would become very problematic.

Using separate booleans for each possible value would be a very dirty workaround, which would significantly affect the user experience when changing the settings. I don't think it's worth it.

Reply:
> What exactly do you whish more clarity about in the description? I described the values as positions, so it should be clear that the lobby is spawned at this position. If I were to really describe this in detail, the space would become very problematic.

`AUTOSETUP_LOBBY_SNAP_OFFSET` is a positive offset when using `GROUND`, a y-coordinate when using `ABSOLUTE` and a negative offset when using `MAX_HEIGHT`. I think this behavior could be confusing and should be clarified.


> Using separate booleans for each possible value would be a very dirty workaround, which would significantly affect the user experience when changing the settings. I don't think it's worth it.

I was thinking about something more like
```yaml
snap:
  type: GROUND
  absoulte:
    ypos: 64
  ground:
    offset: 0
  maxheight:
    offset: 20
```

Reply:
> AUTOSETUP_LOBBY_SNAP_OFFSET is a positive offset when using GROUND, a y-coordinate when using ABSOLUTE and a negative offset when using MAX_HEIGHT. I think this behavior could be confusing and should be clarified.

Ah, I get what you mean. I will try to clarify that in the description. 

> I was thinking about something more like

Now I see what you mean, but that clashes somewhat with the fact that it can only snap to one point at a time.

Reply:
> Now I see what you mean, but that clashes somewhat with the fact that it can only snap to one point at a time.

Yes, but it would allow us to set a different default offset-value for each snap-value as well as making the offset entries somewhat self explanatory. But I agree, it is not really an ideal solution. I'm definitely open for suggestions.

Reply:
> > Now I see what you mean, but that clashes somewhat with the fact that it can only snap to one point at a time.
> 
> Yes, but it would allow us to set a different default offset-value for each snap-value as well as making the offset entries somewhat self explanatory. But I agree, it is not really an ideal solution. I'm definitely open for suggestions.

I thought about it again and came to the conclusion that the solution with the different entries for the different offsets is the nicest way after all. I will implement this soon.